tests/test__plotting.py

```diff
@@ -15,3 +25,140 @@ def test__get_pyplot():
 
     plt_test = test_module._get_pyplot()
     assert plt_test is matplotlib.pyplot
+
+
+def get_report(dlen=100):
+    rng = np.random.default_rng(seed=42)
+    return pd.DataFrame(
+        {
+            "population": dlen * ["default"],
+            "property": pd.Series(dlen * ["value"], dtype="category"),
+            "ids": rng.integers(1, 100, dlen),
+        },
+        index=rng.integers(1, 100, dlen),
+    )
```

Comment:
The mocked (spike) report should be like
```
       ids population
times                
0.1      2    default
0.1      2   default2
0.2      0   default2
```
where:
- the population column should be: categorical
- ids: int64
- times: float64 (as index)

Reply:
Yeah, this was mostly leftovers, I removed it and just `patch`ed `np.concatenate` to return empty array instead.